.github/CODEOWNERS

```diff
@@ -797,6 +797,9 @@
 # ServiceLabel: %Recovery Services Site-Recovery
 # ServiceOwners:                                                   @Sharmistha-Rai
 
+# ServiceLabel: %Recovery Services Data-Replication
```

Comment:
Please move this down into the management library section that starts ~L958, preserving alphabetical ordering by tag.   You'll also need to add the `PRLabel` block scoped to the resource manager directory.    

Please see other management entries for an example.